### ISSUE TITLE
Document annotation: pod-security.kubernetes.io/enforce-policy

### ISSUE THREAD

**Issue by sftim** [kubernetes/website#30809]
**This is a Bug Report**

**Problem:** Pod Security admission sets an annotation, `pod-security.kubernetes.io/enforce-policy`. However, this annotation is not registered / documented.

**Proposed Solution:**
Add that annotation to [Well-Known Labels, Annotations and Taints](https://kubernetes.io/docs/reference/labels-annotations-taints/).

**Page to Update:**
https://kubernetes.io/docs/reference/labels-annotations-taints/

**Kubernetes Version:**
v1.23

**Additional Information:**
/sig auth
/language en
/priority backlog

**Comment by mtardy** [comment]
It piqued my curiosity, and the label was added by [this commit](https://github.com/kubernetes/pod-security-admission/commit/2af9478d1ba2cf23267d020b0c3f3edf54a5a445) (link in the mirror repo). It's an audit annotation that it's returned [in the response](https://github.com/kubernetes/pod-security-admission/blob/2af9478d1ba2cf23267d020b0c3f3edf54a5a445/admission/admission.go#L473) of the admission control. It is only visible in audit logs but I think it is still relevant to be on the cited page.

/assign

**Comment by sftim** [comment]
@mtardy if you're willing to expand this issue to cover all the audit annotations that Pod Security admission sets, that's even better.

**Comment by mtardy** [comment]
Yes, that would be a good idea, I found [the PR that added these annotations](https://github.com/kubernetes/kubernetes/pull/105908). There are three annotations added by this change in the audit logs:
- `pod-security.kubernetes.io/exempt`
- `pod-security.kubernetes.io/audit-violations`
- `pod-security.kubernetes.io/enforce-policy`

I think I can start writing the doc, I explored the commits, the code of the admission and experimented to trigger those annotations in the audit logs to see them act live. Thanks to the super [Pod Security tutorial](https://kubernetes.io/blog/2021/12/09/pod-security-admission-beta/), I was able to start a minikube cluster with the audit logs and the custom AdmissionConfiguration quite fast!
